kolibri/plugins/learn/assets/src/views/content-card-group-carousel/index.vue

```diff
@@ -137,6 +137,15 @@
           'min-width': `${contentCardWidth}px`,
         };
       },
+      buttonTransforms() {
+        const alignmentTransform = 'translate(-50%, -50%)';
+        const mirrorTransform = `scaleX(-1) `;
+
+        return {
+          // must mirror first, order matters
+          transform: (this.isRtl ? mirrorTransform : '') + alignmentTransform,
+        };
+      },
```

Comment:
minor style issue, but this might be better written like:

```javascript
const transforms = ['translate(-50%, -50%)'];
if (this.isRtl) {
  // must mirror first, order matters
  transforms.unshift('scaleX(-1)');
}
return { transform: transforms.join(' ') };
```

doing string concatenation with `+` is brittle because of the spaces.